rts/lra/pom.xml

```diff
@@ -131,9 +134,36 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>${version.checkstyle.plugin}</version>
                 <configuration>
                     <skip>false</skip>
                 </configuration>
+              <dependencies>
+                <dependency>
+                  <groupId>com.puppycrawl.tools</groupId>
+                  <artifactId>checkstyle</artifactId>
+                  <version>${version.checkstyle}</version>
+                </dependency>
+                <dependency>
+                  <groupId>org.wildfly.swarm</groupId>
+                  <artifactId>checkstyle-config</artifactId>
+                  <version>${version.wildfly.swarm.checkstyle}</version>
+                </dependency>
+              </dependencies>
+              <executions>
```

Comment:
Why do we need this when in the main pom we use configLocation: https://github.com/jbosstm/narayana/blob/431f3f759e0630d4a3105c52c678deb83aa5e02e/pom.xml#L381